site/content/en/docs/Examples/custom-controller.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Custom Controller for Agones Game Servers"
+linkTitle: "Custom Controller"
+date:
+publishDate:
+description: >
+  This example shows how to deploy and run the Custom Controller example on Agones, a tool to monitor the running dedicated game servers on Kubernetes.
+---
+
```

Comment:
There should be a brief sentence or two noting prerequisites (running cluster with Agones installed, etc.)

Reply:
prerequisite info is included now

Reply:
Can we move prerequ's out of the page descriptor, and have it's own heading please? It going to look weird up there 😄